src/ui-lib/sass/05-components/02-molecules/00-navigation/_00-nav-menu.scss

```diff
@@ -1,6 +1,7 @@
+$grav-c-nav-menu-link-margin: $grav-sp-s;
```

Comment:
Does this need to be a global SASS variable?

I think you could move it into the `.grav-c-nav-menu { }` block. If you do so, you don't need to use the `grav-...` prefix and could just call it `$nav-link-margin`. Thinking about it, perhaps `$nav-link-padding` is a more appropriate name, since that's what this represents (if I've understood the intent of your code correctly).